final-project/styles/main.scss

```diff
@@ -2,12 +2,826 @@
 
 // test for scss variables
 $primary-color: #2a3fbb;
+$alter-section-color:#F5F8FF;
+$action-button-color:#D2E1FF;
+$nav-text-color: #697CA6;
+$header__logo-size: 26px;
+$header__text-color:#3D4F9F;
+
+@function rem-calc($size) {
+  @return $size / 16px * 1rem;
+}
+
+@mixin tablet-breakpoint {
+  @media only screen and (min-width: rem-calc(768px)) {
+    @content;
+  }
+}
+
+@mixin desktop-breakpoint {
+  @media only screen and (min-width: rem-calc(992px)) {
+    @content;
+  }
+}
 
 * {
   padding: 0;
   margin: 0;
+  padding: 0;
+  margin: 0;
+  font-family: Quicksand;
+  font-variant:100;
 }
 
+button { border:none; } 
+
 h1 {
   color: $primary-color;
+  font-weight: 200;
+
+}
+
+header{
+  display:flex;
+  flex-direction: column;
+  h1{
```

Comment:
Remember that selecting HTML nodes directly is not a good practice, using classes is preferred in case the site structure changes.